src/views/Restoration.vue

```diff
@@ -271,6 +284,27 @@ export default class Restoration extends Vue {
     this.$nextTick(() => this.setHaveUnsavedChanges(false))
   }
 
+  // build applicant orgPerson and assign id (uuid)
+  private parseApplicantOrgPerson (applicant: OrgPersonIF[]): OrgPersonIF[] {
```

Comment:
On line 243, you pass in a single object (applicant[0]). but here you declare the formal parameter as an array.

Reply:
There's still a problem here. On line 247, you pass an array that may have 0 or 1 or more applicants (hey, you never know).

Here (eg, line 294), you treat that array as a single item. JavaScript isn't giving you an error (and may even work if there is a single item in the array) but it's still incorrect code.

I recommend you use "find" to find the first applicant in the parties list, and then only add it to the entity snapshot if "find" found one.

If "find" found no items, how does the rest of the code handle it ... can the user add a new applicant?

Ref: https://www.w3schools.com/jsref/jsref_find.asp

Reply:
Please see comment on line: 245

Reply:
Sorry, I got confused a bit.
- on line 238, you should change applicants -> applicant (since it's an object not an array)
- on line 291, you should change applicant type to OrgPersonIF (since it's an object not an array)

There rest looks OK 👍

Reply:
done